tests/gitea-repositories-meta/user2/lfs.git/info/exclude

```diff
@@ -0,0 +1,6 @@
+# git ls-files --others --exclude-from=.git/info/exclude
+# Lines that start with '#' are comments.
+# For a project mostly in C, the following would be a good set of
+# exclude patterns (uncomment them if you want to use them):
+# *.[oa]
+# *~
```

Comment:
ditto

Reply:
Actually, almost all of the other fixture repos have this file too. So I'm going to leave it. If someone wants to remove them all later, feel free, but for now I'll choose consistency.

Reply:
the two files `description` and `exclude` could be deleted

Reply:
deleted